.github/workflows/exp-updater.yml

```diff
@@ -18,6 +18,10 @@ jobs:
         env:
           AIVEN_TOKEN: ${{ secrets.AIVEN_TOKEN }}
           AIVEN_PROJECT_NAME: ${{ secrets.AIVEN_PROJECT_NAME }}
+      - run: make ./bin/gojson
+        working-directory: ./tools
+      - run: go generate ./...
```

Comment:
> Not sure if it is a good idea, WDYT if we move `//go:generate ...` to `main.go`?
> 
> usually people run `go generate` in the root of the project (and it is convenient to see if there are any generate directive to have them all in one place), and after doing that I found that nothing is happening and the looked it up here.

sorry, but this is a separate Go module and it is a coincidence that we have it in the same repository, thus `go:generate` won't work this way as it will be unaware of the context